views/forms/MultiSelect/index.jsx

```diff
@@ -0,0 +1,47 @@
+import { Autocomplete, Chip, TextField } from '@mui/material';
+import React from 'react';
+
+export const MultiSelect = (props) => {
+  const { field, fieldState } = props;
+  
+  return (
+      <Autocomplete
+        id={props.name}
+        fullWidth={props.fullWidth}
+        getOptionDisabled={(option) => option.isFixed}
```

Comment:
Will we have a a case where a value has stay persistent..? 
Otherwise we can get rid of this

Reply:
This is a cut and paste from the other component, like I stated above.